courses/free-courses-en.md

```diff
@@ -883,7 +885,6 @@
 #### Flask
 
 * [Flask Fridays](https://www.youtube.com/playlist?list=PLCC34OHNcOtolz2Vd9ZSeSXWc8Bq23yEz) - Codemy.com (YouTube)
-* [Flask Tutorials](https://www.youtube.com/playlist?list=PL-osiE80TeTs4UjLw5MM6OjgkjFeUxCYH) - Corey Schafer
```

Comment:
Why remove this if PR is supposed to treat Next.js resources?

Reply:
That's my mistake

Reply:
Reverted by commit 04a439398daad674e1dc821bfa3fc5cac695b1ae